.github/workflows/main.yml

```diff
@@ -149,20 +159,20 @@ jobs:
       - name: Test dmd
         run: ci/run.sh test_dmd
       - name: Test druntime
-        if: '!matrix.coverage'
+        if: '!matrix.coverage && (success() || failure())'
```

Comment:
why was this needed and how does this help? (seems to add  `&& true` which is a no-op?)

Reply:
It runs all of the test steps (if all steps before were successful), doesn't abort after the 1st failing test step. I.e., you see all failures and don 't have to fix the compiler testsuite before seeing results for druntime and Phobos etc.